Classes/Model/Download.cs

```diff
@@ -700,11 +702,16 @@ public static int Compare(int epid1, int epid2)
                         case DownloadCols.Duration:
                             orderBy = "duration" + (sortAsc ? string.Empty : " desc");
                             break;
+                        case DownloadCols.ProgrammeName:
+                            // OrderBy Programme.Name
+                            sqlCmd = "select downloads.epid from downloads, episodes, programmes where downloads.epid=episodes.epid and episodes.progid=programmes.progid order by ";
+                            orderBy = "programmes.name" + (sortAsc ? string.Empty : " desc");
+                            break;
                         default:
                             throw new InvalidDataException("Invalid column: " + sortBy.ToString());
                     }
 
-                    using (SQLiteCommand command = new SQLiteCommand("select downloads.epid from downloads, episodes where downloads.epid=episodes.epid order by " + orderBy, FetchDbConn()))
+                    using (SQLiteCommand command = new SQLiteCommand(sqlCmd + orderBy, FetchDbConn()))
```

Comment:
Although it would be slightly less efficient, I'd say just add the programmes table to the original SQL command which then saves the need for the `sqlCmd` variable and duplicating parts of the statement.  I'm pretty sure we won't be able to notice the improvement in performance when the `Programme Name` column isn't shown, even on huge lists of downloads.